site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "Set up model workflows"
+date: last-modified
+---
+
+Workflows can be configured to match your organizational needs for overseeing model development, validation, or implementation activities.
+
+Default workflows are provided for you as suggestions. You can customize the following default workflows:
+
+- **Model documentation:** When documentation is created
+- **Inventory model:** When the model is created
+
+<!-- 
+- Model validation workflow is in missing as of May 2024
+- These default workflows will eventually be replaced by a trigger you can define
+ -->
+
+ Setting up your workflow involves two major steps:
+
+ 1. [Configuring workflow steps](#configure-workflow-steps) 
+ 2. [Linking the workflow together](#link-workflow-together)
+
+
+ <!-- The following is no longer true as of May 31, 2024: -->
+
+<!-- ::: {.callout-warning title="Adjustments to workflows are not retroactive."}
+When a workflow is edited, the new flow will only apply to new models created after the adjustment. 
+::: -->
+
+## Prerequisites
+
+- To customize workflows, you must hold the `Customer Admin` or `Developer` role
+- [Resource statuses](customize-resource-statuses.qmd) must be fully configured
+
+## Configure workflow steps
+
+To add or edit workflow steps:
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. In the left sidebar, click [**{{< fa gear >}} Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workplace Settings, select **{{< fa arrow-right-arrow-left >}} Workflows**.
+
+4. Under available workflows, select the process you'd like to add or edit the steps for:
+ 
+    - To add a step, drag and drop a new step onto the canvas. Hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears, then click on it and select **{{< fa gear >}} Configure**.
+    - To edit a step, hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears. Then click on it and select **{{< fa gear >}} Configure**.
+
+5. After you're finished with step configuration, click **Save** to apply your changes before closing the configuration panel with **{{< fa x >}}**. 
+
+    ![](configure-steps.gif){fig-alt="A gif workflow step configuration."}
+
+    After you've configured a step, you can then [link your workflow together](#link-workflow-together).
+
+::: {.callout-tip appearance="minimal"}
+Refer to **[Available step types](#available-step-types)** for information on available step types and details on how to set up conditional requirements.
```

Comment:
GLOBAL: I would not bold links to subsections. This topic already has a ton of highlighting in it, so whatever extra visual clutter we can omit, we should.